libnavigation-core/src/main/java/com/mapbox/navigation/core/adas/AdasisDataSendingConfig.kt

```diff
@@ -12,6 +12,8 @@ import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
  * @param enableRetransmission if true, retransmission will be enabled
  * (package will be appended with retransmission data, messages from previous cycles)
  * @param retransmissionMeters after passing this distance, messages will not be retransmitted
+ * @param treeTrailingLength the trailing length of the path tree, relatively
```

Comment:
relative to

Reply:
I don't consider ChatGPT to be the true source of grammar, but:
<img width="819" alt="image" src="https://github.com/mapbox/mapbox-navigation-android/assets/32109537/33122598-67a5-4918-b42b-30e6bdb28ff2">